settings.py

```diff
@@ -0,0 +1,3 @@
+myHost = '127.0.0.1'
+myPort = 8050
+timeScale = 1 # in days
```

Comment:
Good idea! Make it default to 2? That's what it says in the docs now...

Reply:
yes of course !